### ISSUE TITLE
how  to  convert  predict-val of  "python-format" catboost model    to probability

### ISSUE THREAD

**Issue by dicomcmshow** [catboost/catboost#2007]
Problem: If run this "model.save_model(r'catboost.py',format='python')",can save model to python file, when load the model to predict data ,return me a value which is not a probability ,I want to konw how to convert the return result  to probability
catboost version:1.0.4
Operating System:win7
CPU:i53470
GPU:no

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
raw formula values are converted to probabilities using sigmoid
https://catboost.ai/en/docs/concepts/output-data_model-value-output#classification

**Comment by dicomcmshow** [comment]
> raw formula values are converted to probabilities using sigmoid https://catboost.ai/en/docs/concepts/output-data_model-value-output#classification
Thank you very much.  but I find  the value of  'using sigmoid with  the return-val of python model'  is different with the result of cbm model predict-val   .and  different with the pmml model predict-val,  so I am very puzzled now：different model  different result,  which is the right val

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
hm, this looks like a bug.
could you please provide a reproducer?

**Comment by Anadorr** [comment]
I had the same problem - for multiclass classification with MultiClass loss function, you should apply softmax and not sigmoid to get probabilities. [Code.](https://github.com/catboost/catboost/blob/master/catboost/libs/eval_result/eval_helpers.cpp#L362-L363)

**Comment by andrey-khropov** [comment]
> hm, this looks like a bug. could you please provide a reproducer?

No additional feedback has been provided, so I'm closing this issue. Feel free to reopen with more details if the problem still persists.
